### PR TITLE
Adding audit-log plugin skelton

### PR DESCRIPTION
**Pull request by Lathaguna**
Hello @jvz ,

The pom.xml generated from the mvn command uses the items mentioned in the tutorial. I notice it by default uses MIT license. Should I change it to `Apache License v2.0` as you were discussing with @bitwiseman in the gitter channel?

Thanks

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.33</version>
+        <relativePath />
+    </parent>
+    <artifactId>audit-log-plugin</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>2.7.3</jenkins.version>
```
Comment (jvz) [inline pom.xml]: You can upgrade this to `2.60.3` for a Java 8 baseline, though we may end up having to upgrade that to the latest Jenkins version if we rely on new APIs.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    </properties>
+    <name>Audit Log Plugin</name>
+    <description>Audit logging for Jenkins with predefined audit events</description>
+    <url>https://wiki.jenkins-ci.org/display/JENKINS/TODO+Plugin</url>
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
```
Comment (jvz) [inline pom.xml:23]: I'd be alright with using the MIT license. Just make sure to swap out the LICENSE file with the MIT license as well.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.33</version>
```
Comment (jvz) [inline pom.xml]: Oh wow, the archetype uses an ancient parent version. Take a look at https://github.com/jenkinsci/plugin-pom for some more docs. We should use version 3.25 and need to add `<java.level>8</java.level>` to the `<properties>` section below.

**Review by jvz — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just a couple modifications to get us up to date with the latest versions of things and we're good to go.

File: pom.xml (hunk 2 above)
Comment (Lathaguna) [reply pom.xml:23]: Yes. I updated the LICENSE file. The [yyyy] and [name of copyright owner] fields are not changed as it was in the old LICENSE. I hope that is fine.

**Comment by Lathaguna** [comment]
Thanks @jvz for the comments. I updated the versions as suggested. Please have a look while you get time. 

I shall now continue adding the catalog file.

File: pom.xml (hunk 2 above)
Comment (jvz) [reply pom.xml:23]: We can update the copyright later, no worries!

**Review by jvz — APPROVED** [review]

**Pull request merged**